samples/boards/nrf52/temp/src/main.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2019 Aaron Tsui <aaron.tsui@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <sensor.h>
+#include <misc/printk.h>
+
+void main(void)
+{
+	struct sensor_value temp;
+
+	struct device *dev = device_get_binding("TEMP_0");
```

Comment:
CONFIG_TEMP_NRF5_NAME is defined for the binding name.

Reply:
Thank you for reviewing. I will change it to 'CONFIG_TEMP_NRF5_NAME'.